### PR TITLE
Improve JSON attributes template example

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Improve JSON attributes template example

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by emontnemery** [comment]
@deepcoder Please help to review

File: source/_integrations/sensor.mqtt.markdown [hunk 1]
```diff
@@ -173,9 +177,9 @@ sensor:
 
 The state and the attributes of the sensor by design do not update in a synchronous manner if they share the same MQTT topic. Temporal mismatches between the state and the attribute data may occur if both the state and the attributes are changed simultaneously by the same MQTT message. An automation that triggers on any state change of the sensor will also trigger both on the change of the state or a change of the attributes. Such automations will be triggered twice if both the state and the attributes change. Please use a [MQTT trigger](/docs/automation/trigger/#mqtt-trigger) and process the JSON in the automation directly via the {% raw %}`{{ trigger.payload_json }}`{% endraw %} [trigger data](/docs/automation/templating/#mqtt) for automations that must synchronously handle multiple JSON values within the same MQTT message.
 
-### Get battery level
+### Value template example - Owntracks battery level sensor
```
Comment (fabaff) [inline source/_integrations/sensor.mqtt.markdown]: We are in the "Examples" section.

```suggestion
### Owntracks battery level sensor
```

File: source/_integrations/sensor.mqtt.markdown [hunk 2]
```diff
@@ -195,7 +199,7 @@ sensor:
 ```
 {% endraw %}
 
-### Get temperature and humidity
+### Value template example - Temperature and humidity sensors
```
Comment (fabaff) [inline source/_integrations/sensor.mqtt.markdown]:
```suggestion
### Temperature and humidity sensors
```

**Review by fabaff — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**